src/FSharpSource.targets

```diff
@@ -230,6 +233,7 @@
     <DefineConstants>$(DefineConstants);NO_LOADER_OPTIMIZATION</DefineConstants>
     <DefineConstants>$(DefineConstants);SILVERLIGHT_COMPILER_FSHARP_CORE</DefineConstants>
     <DefineConstants>$(DefineConstants);I_DONT_KNOW_HOW_TO_DO_THIS_YET</DefineConstants>
+    <DefineConstants>$(DefineConstants);NETSTANDARD1_5</DefineConstants>
```

Comment:
the generated file from fssrgen contains an `#if NETSTANDARD1_5` so the same file can work for both net46 and netstandard1.5 (dnxcore50 too, but now is useless)
`NETSTANDARD1_5` it's the default define added for `netstandard1.5` framework (like `NET46` for .net 4.6)